File: tournaments/rightmostRoundNumber/rightmostRoundNumber.py
def rightmostRoundNumber(inputArray):
    x = 0
    n = 0
    f = False
    
    while len(inputArray) > x:
        if inputArray[x] % 10 == 0:
            n = x
            f = True
        x += 1
    if f:
        return n
    return -1

File: tournaments/rightmostRoundNumber/rightmostRoundNumber.cpp
int rightmostRoundNumber(std::vector<int> inputArray) {

  int ans = -1;
  for (int i = 0; i < inputArray.size(); i++) {
    if (inputArray[i] % 10 == 0) {
      ans = i;
    }
  }

  return ans;
}
